docs/source/faqs-new.md

```diff
@@ -0,0 +1,40 @@
+(faqs-new)=
+# Frequently Asked Questions (FAQs)
+Below are some common questions and answers regarding our HPC cluster and its operation.
+
+::::{dropdown} My OOD job shuts down quickly before/during starting?

```

Comment:
My OOD Desktop job starts and quickly moves to `Completed` without starting. (I think this is the ticket and we can double check in SNOW).